### PR TITLE
Tth to tau tau 2015 feb28

### PR DESCRIPTION
**Pull request by veelken**
The pull request adds information specific to the ttH, H -> tautau analysis to the VHbb Ntuple

### CODE REVIEW

File: VHbbAnalysis/Heppy/python/tthobj.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (arizzi) [inline VHbbAnalysis/Heppy/python/tthobj.py]: you can include this in vhbbobj.py

**Comment by veelken** [comment]
@ arizzi Ok, done.

**Comment by arizzi** [comment]
it seems this PR is not mergable... can you rebase?

**Comment by arizzi** [comment]
btw I'm fixing the mergability issue and implementing what I suggested in #13 I close these two PRs and I'll open a new one

**Pull request closed without merging**